src/styles/globals.scss

```
@@ -1246,3 +1246,9 @@
     gap: pxToRem(4px);
   }
 }
+
+.FieldNote {
+  &--error {
+    word-break: break-all;
```

Can we try if `break-word` works here? We had an issue with `break-all` in the notification, where it broke a word in the middle.

```suggestion
    word-break: break-word;
```